src/Core/Style.js

```diff
@@ -160,6 +161,64 @@ function defineStyleProperty(style, category, name, value, defaultValue) {
     style[category][name] = value;
 }
 
+/**
+ * @class
+ * @classdesc StyleContext stores metadata of one FeatureGeometry that are needed for its style computation:
+ * type of feature and what is needed (fill, stroke or draw a point, etc.) as well as where to get its
+ * properties and its coordinates (for base_altitude).
+ *
+ * @property {Coordinates}      worldCoord @private Coordinates of the FeatureGeometry in world system.
```

Comment:
wording: `Feature coordinates in world space`

Reply:
I changed the doc to move all private at the end as this properties are not really the coordinates of the point but intermediate obect to get local or worl coordinates.

Reply:
yeys it's better thanks